<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSubLevelWindow</class>
 <widget class="QWidget" name="CSubLevelWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>648</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>648</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>648</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="window_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>22</y>
     <width>440</width>
     <height>34</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>440</width>
     <height>34</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>440</width>
     <height>34</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
font-size:15pt;
color:gray;
border: 0px; 
background-color: black;
} </string>
   </property>
   <property name="text">
    <string>   Setting</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>94</y>
     <width>220</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>72</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>220</width>
     <height>72</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background: rgb(6,17,35);
font-size:15pt;
color:gray;
border-left:1px solid balck;border-top:1px solid black;border-right:1px solid rgb(8,24,57);border-bottom:1px solid rgb(8,24,57);
outline: none;
} </string>
   </property>
   <property name="text">
    <string>Brightness</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>94</y>
     <width>220</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>72</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>220</width>
     <height>72</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background: rgb(6,17,35);
font-size:15pt;
color:gray;
border-left:1px solid balck;border-top:1px solid black;border-right:1px solid rgb(8,24,57);border-bottom:1px solid rgb(8,24,57);
outline: none;
} </string>
   </property>
   <property name="text">
    <string>Train running num</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>166</y>
     <width>220</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>72</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>220</width>
     <height>72</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background: rgb(6,17,35);
font-size:15pt;
color:gray;
border-left:1px solid balck;border-top:1px solid black;border-right:1px solid rgb(8,24,57);border-bottom:1px solid rgb(8,24,57);
outline: none;
} </string>
   </property>
   <property name="text">
    <string>Enter DATA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>166</y>
     <width>220</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>72</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>220</width>
     <height>72</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background: rgb(6,17,35);
font-size:15pt;
color:gray;
border-left:1px solid balck;border-top:1px solid black;border-right:1px solid rgb(8,24,57);border-bottom:1px solid rgb(8,24,57);
outline: none;
} </string>
   </property>
   <property name="text">
    <string>Driver ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>238</y>
     <width>220</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>72</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>220</width>
     <height>72</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background: rgb(6,17,35);
font-size:15pt;
color:gray;
border-left:1px solid balck;border-top:1px solid black;border-right:1px solid rgb(8,24,57);border-bottom:1px solid rgb(8,24,57);
outline: none;
} </string>
   </property>
   <property name="text">
    <string>Function Test</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
